openhtf/core/test_state.py

```diff
@@ -426,7 +426,7 @@ def _asdict(self):
         'name': self.name,
         'codeinfo': self.phase_record.codeinfo,
         'descriptor_id': self.phase_record.descriptor_id,
-        'start_time_millis': long(self.phase_record.start_time_millis),
+        'start_time_millis': int(self.phase_record.start_time_millis),
```

Comment:
This code will behave subtly different on Python2 if the value is > 32 bits.
I think adding `from past.builtins import long` at the top and reverting to long is the right thing here. (2to3 did this)